react/week3/todo-week3/src/timer.js

```diff
@@ -0,0 +1,14 @@
+import React, { useState, useEffect } from "react"
+
+// Counter
+function Timer() {
+  const [TimerState, setTimerState] = useState(0)
```

Comment:
When you capitalize a variable in React, that means it's a component. 
This should be `const [timer, setTimer] = useState(0)`